parm/soca/obs/config/sst_metopa_l3u.yaml

```diff
@@ -0,0 +1,46 @@
+obs space:
+  name: sst_metopa_l3u
```

Comment:
@guillaumevernieres, is it worth renaming this as: `sst_avhrr_metopa_l3u`, similar to `sst_abi_g16_l3c` and `sst_viirs_n20_l3u`, to follow a pattern?

Reply:
I like this concept

Reply:
I agree @ShastriPaturi , it should be renamed.